sqflite/lib/src/sentry_batch.dart

```diff
@@ -64,14 +70,24 @@ class SentryBatch implements Batch {
 
         span?.status = SpanStatus.ok();
 
+        breadcrumb.data?['status'] = 'ok';
+
         return result;
       } catch (exception) {
         span?.throwable = exception;
         span?.status = SpanStatus.internalError();
 
+        breadcrumb.data?['status'] = 'internalError';
+        breadcrumb = breadcrumb.copyWith(
+          type: 'error',
```

Comment:
type shouldn't be `error` here

Reply:
I checked in the documentation and error seemed appropriate: https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types

Do you have a suggestion which type to use instead?

Reply:
my understanding is the `error` type is for Exceptions.
Looking at what is done for [Java OkHttp](https://github.com/getsentry/sentry-java/blob/12704a922bb5b964a9f2f88753acffaeee85ff14/sentry-android-okhttp/src/main/java/io/sentry/android/okhttp/SentryOkHttpInterceptor.kt#L142), we send a breadcrumb with type `http`, regardless the status code.
Let me ask the team and get back to you

Reply:
Thank you for clearing this up! Warning seems better yes, as users might handle these.

Reply:
Ok updated them to query. Thinking if this is also correct in `apply` and `commit`, which are write operations. 🤔

Reply:
@stefanosiano any thoughts on that? not sure about that here

Reply:
yeah, I think it's fine, as they are still operations on the database